signal_scanner_bot/filters.py

```diff
@@ -61,24 +61,11 @@ def _f_not_recent(data: Dict) -> bool:
 ################################################################################
 # Twitter Filters
 ################################################################################
-def _f_retweeted(status: Status) -> bool:
-    # Status is retweeted
-    return status.retweeted
-
-
-def _f_retweet_text(status: Status) -> bool:
-    # Status text starts with "RT @"
-    # Twitter uses that to identify a retweet
-    return status.text.startswith("RT @")
```

Comment:
I think we may still need this filter - this one filters out bare retweets (not quote tweets). For whatever reason, the Twitter track API registers bare retweets as a "new status" or something similar. So I think we'll need to keep this to prevent trusted tweeter's retweets from being posted in signal.